`make` reports an error but I did not yet figure out what is going on: 
```
Building HTML documentation.
/cygdrive/c/projects/julia/usr/bin/julia.exe `cygpath -w /cygdrive/c/projects/julia/doc/make.jl` > docbuild.log 2>&1
make[2]: *** [Makefile:34: html] Error 1
make[2]: Leaving directory '/cygdrive/c/projects/julia/doc'
make[1]: *** [Makefile:119: docs] Error 2
make[1]: Leaving directory '/cygdrive/c/projects/julia'
```

--- a/base/docs/basedocs.jl
+++ b/base/docs/basedocs.jl
@@ -1275,7 +1275,7 @@ Any
 
 `Union{}`, the empty [`Union`](@ref) of types, is the type that has no values. That is, it has the defining
 property `isa(x, Union{}) == false` for any `x`. `Base.Bottom` is defined as its alias and the type of `Union{}`
-is [`Core.TypeofBottom`](@ref).
+is `Core.TypeofBottom`.
 
 # Examples
 ```jldoctest
@@ -1386,6 +1386,6 @@ field names; fields are only accessed by index.
 
 See the manual section on [Tuple Types](@ref).
 """
-kw"Tuple"
+Tuple
 
 end

--- a/doc/src/stdlib/base.md
+++ b/doc/src/stdlib/base.md
@@ -83,7 +83,7 @@ Base.widen
 Base.identity
 ```
 
-## Types
+## Dealing with Types
 
 ```@docs
 Base.supertype
@@ -109,9 +109,20 @@ Base.isbits
 Base.isleaftype
 Base.typejoin
 Base.typeintersect
-Base.Val
-Base.Enums.@enum
 Base.instances
+```
+
+## Special Types
+
+```@docs
+Core.Any
+Base.Enums.@enum
+Core.Union
+Union{}
+Core.UnionAll
+Core.Tuple
+Base.Val
+Core.Vararg
 ```
 
 ## Generic Functions

